geometry_msgs/Pose goal
---
uint8 distance
